ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -130,8 +130,9 @@ case class FieldStats(field: String,
 }
 
 /** Big diff between two data sets given a primary key. */
-class BigDiffy[T : Coder](lhs: SCollection[T], rhs: SCollection[T],
-                          diffy: Diffy[T], keyFn: T => MultiKey, ignoreNan: Boolean = false) {
+class BigDiffy[T : Coder](@transient val lhs: SCollection[T], @transient val rhs: SCollection[T],
+                          diffy: Diffy[T], keyFn: T => MultiKey, ignoreNan: Boolean = false)
+  extends Serializable {
```

Comment:
Does it work without this line but with the change on line 177?

Reply:
I tried removing `extends Serializable` from both and rerunning; it failed.

Reply:
I believe only `case` classes are serializable by default (and have registered coders built-in in scio), hence the need to explicitly mark this class as `Serializable`. Great job find that out @anne-decusatis !